godeltaprof/svcinfo/svcinfo.go

```diff
@@ -0,0 +1,67 @@
+package svcinfo
+
+import (
+	"os"
+	"runtime"
+	"runtime/debug"
+	"sync"
+)
+
+const (
+	LabelNameServiceRepository = "service_repository"
```

Comment:
I don't think you need this.

Reply:
I was going to add both mapping and labels in one PR, but decided to separate